sim/target-agnostic.mk

```diff
@@ -164,11 +173,9 @@ $(PLATFORM) = $(OUTPUT_DIR)/$(DESIGN)-$(PLATFORM)
 $(PLATFORM): $($(PLATFORM))
 
 .PHONY: driver
-driver: $($(PLATFORM))
-
-fpga_dir = $(firesim_base_dir)/../platforms/$(PLATFORM)/aws-fpga
+driver: $(PLATFORM)
 
-$(f1): export CXXFLAGS := $(CXXFLAGS) $(common_cxx_flags) $(DRIVER_CXXOPTS) -I$(fpga_dir)/sdk/userspace/include
+$(f1): export CXXFLAGS := $(CXXFLAGS) $(common_cxx_flags) $(DRIVER_CXXOPTS)
```

Comment:
this is okay to remove?

Reply:
This was just moved to a better location: https://github.com/firesim/firesim/blob/06088672fd542645e638b7fe73def744c1dbac07/sim/target-agnostic.mk#L46